cobalt/browser/h5vcc_system/h5vcc_system_impl.cc

```diff
@@ -25,6 +31,25 @@ using starboard::android::shared::StarboardBridge;
 
 namespace h5vcc_system {
 
+namespace {
+
+#if !BUILDFLAG(IS_ANDROID)
```

Comment:
I think there's the case of Android _and_ Evergreen (AOSP) in which we have and use Starboard stuff (for more than the SbPlayer). So it's better to use `#if BUILDFLAG(USE_EVERGREEN)` (or similar, @andrewsavage1 would know exactly the enchantment) to identify platforms with full Starboard functionality. An added value is that it's also positive logic, which is easier to parse.